src/blocks/recommendations/components/parsely-recommendations-inspector-controls.tsx

```diff
@@ -40,23 +39,23 @@ const ParselyRecommendationsInspectorControls = ( {
 				<TextControl
 					label={ __( 'Title', 'wp-parsely' ) }
 					value={ title }
-					onChange={ useCallback( ( value: string ): void => setAttributes( { title: value } ), [ title ] ) }
```

Comment:
We added these `useCallback` while implementing the feedback of SonarCloud but after the upgrade we have to remove it because eslint show warnings that these are extra and SonarCloud also don't show any issue on these now.